extensions/devservices/deployment/src/main/java/io/quarkus/devservices/deployment/DevServicesProcessor.java

```diff
@@ -213,10 +213,10 @@ private static String[] getNetworks(Container container) {
         return networks.entrySet().stream()
                 .map(e -> {
                     List<String> aliases = e.getValue().getAliases();
-                    if (aliases == null) {
+                    if (aliases == null || aliases.isEmpty()) {
```

Comment:
This is just for safety as the contract might change given it's considered bad practice to return a null list.